common/api-review/remote-config-exp.api.md

```diff
@@ -0,0 +1,48 @@
+## API Report File for "@firebase/remote-config-exp"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { FirebaseApp } from '@firebase/app-types-exp';
+import { LogLevel } from '@firebase/remote-config-types-exp';
+import { RemoteConfig } from '@firebase/remote-config-types-exp';
+import { Value } from '@firebase/remote-config-types-exp';
+
+// @public (undocumented)
+export function activate(remoteConfig: RemoteConfig): Promise<boolean>;
+
+// @public (undocumented)
+export function ensureInitialized(remoteConfig: RemoteConfig): Promise<void>;
+
+// @public (undocumented)
+export function fetchAndActivate(remoteConfig: RemoteConfig): Promise<boolean>;
+
+// @public (undocumented)
+export function fetchConfig(remoteConfig: RemoteConfig): Promise<void>;
+
+// @public (undocumented)
+export function getAll(remoteConfig: RemoteConfig): Record<string, Value>;

```

Comment:
I'm unfamiliar w `Record`. The return type before was `object`:
https://github.com/firebase/firebase-js-sdk/blob/master/packages/remote-config-types/index.d.ts#L68 -> https://firebase.google.com/docs/reference/js/firebase.remoteconfig.RemoteConfig#getall

Using `{ [key: string]: Value }` still seems idiomatic: 
https://www.typescriptlang.org/docs/handbook/interfaces.html#indexable-types

What's the value of using `Record`?

Reply:
It's an alias of `{ [key:string]: Value}`. The only reason to use it is because it is shorter.

Reply:
👍  Good to know. Thx